packages/input-tel-dropdown/src/LionInputTelDropdown.js

```diff
@@ -227,6 +227,9 @@ export class LionInputTelDropdown extends LionInputTel {
     this._onDropdownValueChange = this._onDropdownValueChange.bind(this);
     /** @type {EventListener} */
     this.__syncRegionWithDropdown = this.__syncRegionWithDropdown.bind(this);
+
+    /** Set to true, since the input-tel will set the correct activeRegion */
+    this.hasNoDefaultSelected = true;
```

Comment:
This was a prop of RichSelect, right?
Do you want to pass it along or smth, since it's not on InputTel prototype?
And if so, what's the goal of not setting/showing the default in the dropdown?

Reply:
It's a property of listbox.

Without this the input and dropdown will both define their own modelValue, and then decide together which one wins. This goes correct in lion, but not in our extended layer.

By setting it default to none, les code needs to run and it also prevents bugs (as in our extended layer).

Reply:
But then it would need to be applied on refs.dropdown?

Reply:
Let me double check

Reply:
Removed it again